### PR TITLE
make test_e2e_asr_mulenc.py faster 

### PR DESCRIPTION
**Pull request by ruizhilijhu**
(local run time 152.75s -> 8.86s)

1. decreased the feature size and model size
2. created a separate test module for ctc plot and att plot
3. deleted/merged redundant test cases

### CODE REVIEW

File: test/test_e2e_asr_mulenc.py [hunk 1]
```diff
@@ -542,55 +312,17 @@ def test_model_trainable_and_decodable(module, num_encs, model_dict):
...
-    plot = PlotCTCReport(
-        model.calculate_all_ctc_probs, batchset[0], tmpdir, None, None, None
-    )
-    if args.mtlalpha > 0:
-        for i in range(num_encs):
-            # ctc-encoder
-            plot._plot_and_save_ctc(ctc_probs[i][0], "{}/ctc{}.png".format(tmpdir, i))
-
     # test decodable
     with torch.no_grad(), chainer.no_backprop_mode():
```
Comment (ShigekiKarita) [inline test/test_e2e_asr_mulenc.py:320]: Can you delete all "chainer" lines? I think all of them are useless.

**Comment by ShigekiKarita** [comment]
8.86 is awesome!!

File: test/test_e2e_asr_mulenc.py (hunk 1 above)
Comment (ruizhilijhu) [reply test/test_e2e_asr_mulenc.py]: Agreed and done!

**Review by ShigekiKarita — APPROVED** [review]
Thanks!! Now `test python` (7:58) is faster than `install espnet` (9:00) at Circle CI ubuntu18 https://app.circleci.com/pipelines/github/espnet/espnet/6305/workflows/28d2669a-b537-47bb-8980-cdd3573e5fe3/jobs/21255

**Pull request merged**